### ISSUE TITLE
Manifest not fully updated?

### ISSUE THREAD

**Issue by HeliosStudios** [Bungie-net/api#67]
Hi 

I'm just wondering if there is any clarification regarding if/when the manifest will match the in-game inventory status. I've been working on OBS and Twitch Extension overlays, and using my own Titan in testing has led to at least 3 weapons i have not existing in the InventoryItem table of the latest copy of the mobile database

When looking on DIM the items come up as classified?
Just wondering if its a small delay on an updated manifest, as it was making me concerned during testing
Any clarification would be great

Thanks
Barry

**Comment by HeliosStudios** [comment]
Hi Thornheart

I had weapons such as the Minuet-42, Acantha-D grenade launcher and Nergal from the beta, but honestly its no big panic, if they are coming that's all that really matters, just was concerned in testing. I can work on the overlay features using rares etc until we get a new manifest push, i'm sure you guys have more pressing things than this. Thank you for replying soo quickly

Regards
Barry

**Comment by vthornheart-bng** [comment]
Oh - you're seeing weapons from the beta when you make platform calls?  (I may be misunderstanding, feel free to correct me!)

Aye, our goal is for these to all eventually be un-blacklisted once the designers allow us to - but as in D1, sometimes they drag their feet and we have to kick them once someone brings up that you can indeed already get the item in the game!

**Comment by HeliosStudios** [comment]
Yes that's it i've already got them in-game but when i use their hash IDs against the manifest they don't return the usual name, icon info etc
Just seemed odd being able to get them in-game and then not find them within the manifest, which was what had me concerned during testing as i use the SQLite database to enhance performance on requests
Hopefully they will allow un-blacklisting soon as i'm handling exceptions atm with a generic 'Coming Soon' icon

**Comment by vthornheart-bng** [comment]
Yeah - I'll let them know, but definitely make sure your app is prepared to serve people Classified items - because as they un-blacklist some, they'll inevitably blacklist more for future releases :(  It's unfortunately an ongoing thing that all devs will have to handle in their apps when they run into something blacklisted.

**Comment by HeliosStudios** [comment]
Yea i will do that for sure, i was only expecting the Raid gear to be locked down in advance of it dropping but having worked in game industry 12 years its not first time i've seen this in practice, like you say its unfortunate but we can work around it. I'm off to patch this with a permanent fix in that case moving forward, many thanks again for the advise and info

Regards
Barry

**Comment by vthornheart-bng** [comment]
Just for reference, could you send me the hash identifiers of items you're seeing in your inventory that are being blocked?  Just the hash (uint) would be useful so I can give them some examples.

**Comment by HeliosStudios** [comment]
3778520450 - Halfdan-D Auto Rifle - Kinetic
3860697508 - Minuet 42 Hand Cannon - Energy

Those are 2 quick ones from those i still have atm Thornheart, on DIM they appear as blank icons and the Inspect console on Chrome returns - This item is not in the current manifest and will be added at a later time by Bungie

That may be DIMs response but could be on Bungie end, not sure how DIM process that
